numba_cuda/numba/cuda/testing.py

```diff
@@ -177,6 +177,8 @@ def assertFileCheckMatches(
 
             self.fail(
                 f"FileCheck failed:\n{matcher.stderr.getvalue()}\n\n"
+                + f"Check prefixes:\n{check_prefixes}\n\n"
```

Comment:
I think this was removed in #366 - is this added back here by mistake, or because removing it was a mistake?

When I saw it removed in #366, I assumed that was fine because it makes the test output a little more concise, and the prefixes and patterns could always be observed in the dumps if one is interested. Was that a reasonable evaluation?